lesson_13/bank/bank_app/src/main/java/com/codedifferently/lesson13/bank/SavingsAccount.java

```diff
@@ -0,0 +1,15 @@
+package com.codedifferently.lesson13.bank;
+
+import java.util.Set;
+
+public class SavingsAccount extends CheckingAccount {
```

Comment:
A `CheckingAccount` could be considered a type of `SavingsAccount`, but not the other way around.